android-sdk/src/main/java/org/sagebionetworks/bridge/android/util/retrofit/RxUtils.java

```diff
@@ -14,11 +15,20 @@
  */
 @AnyThread
 public class RxUtils {
+
+    /**
+     * Converts a Call to a Single.
+     *
+     * @param call call which returns Response with body type T
+     * @return Single created from call, which emits T
+     * @typeparam T type returned by Single and Call
+     */
     @NonNull
     public static <T> Single<T> toBodySingle(@NonNull Call<T> call) {
         checkNotNull(call);
 
-        return Observable.create(new BodyOnSubscribe<>(new CallOnSubscribe<>(call))).toSingle();
+        return Observable.create(new BodyOnSubscribe<>(new CallOnSubscribe<>(call)))
+                .subscribeOn(Schedulers.io()).toSingle();
```

Comment:
This makes all Retrofit Call<T> happen on the io(). That way we won't have to think about thread every time we do a network call